lib/Qpsmtpd/Plugin.pm

```diff
@@ -347,10 +347,16 @@ sub _register_standard_hooks {
     }
 }
 
+sub db_args {
+    my ( $self, %arg ) = @_;
```

Comment:
should we die if the wrong number of args are passed, or just warn?  if all we want is to warn, perl already does this for us:

```
perl -Mstrict -wle 'my %h = (1,2,3)'
Odd number of elements in hash assignment at -e line 1.
```